bin/build_container.sh

```diff
@@ -1,53 +0,0 @@
-#!/bin/bash
-
-set -e
-
-THIS_DIR=$(dirname $(readlink -f $0))
-PROJ_ROOT=${THIS_DIR}/..
-BASE_DIR=${PROJ_ROOT}/../..
-
-IMAGE_NAME="experiment-lammps"
-VERSION="$(< ${BASE_DIR}/VERSION)"
-FAASM_VERSION="$(< ${BASE_DIR}/faasm/VERSION)"
-
-pushd ${PROJ_ROOT} >> /dev/null
-
-export DOCKER_BUILDKIT=1
-
-# Docker args
-if [ "$1" == "--no-cache" ]; then
-    NO_CACHE=$1
-else
-    NO_CACHE=""
-fi
-
-# Create faasm.ini file for up-to-date knative deployment
-${BASE_DIR}/faasm/bin/knative_route.sh | tail -7 > faasm.ini
-
-docker build \
-    ${NO_CACHE} \
-    -t faasm/${IMAGE_NAME}:${VERSION} \
-    -f ${PROJ_ROOT}/Dockerfile \
-    --build-arg EXPERIMENTS_VERSION=${VERSION} \
-    ${PROJ_ROOT}
-
-# Remove faasm.ini file
-rm faasm.ini
-
-# Upload lammps function + data
-docker run --rm \
-    faasm/experiment-lammps:${VERSION} \
-    inv -r faasmcli/faasmcli upload lammps main \
-    /code/experiment-lammps/third-party/lammps/install/bin/lmp
-docker run --rm \
-    faasm/experiment-lammps:${VERSION} \
-    inv -r faasmcli/faasmcli state.shared-file \
-    /data/in.controller /lammps-data/in.controller
```

Comment:
This script wasn't just building the container, it was uploading data as well. This was misleading given the name of the script and should be in a stand-alone task.